### ISSUE TITLE
Symbols with mask crop child layers

### ISSUE THREAD

**Issue by koenbok** [bomberstudios/sketch-framer#78]
If you import these to framer the kermit in the ContainerA will be the right unchopped size and the other one will be as high as ContainerB (flattened).

![image](https://cloud.githubusercontent.com/assets/206779/4408549/d5e011ca-44d4-11e4-8134-a3bf8fe276d3.png)

**Comment by benjaminnathan** [comment]
Okay, can reproduce this. It's a scrolling issue with symbols.

Sketch + Framer File: http://cl.ly/Zmna

#### Case
1. Two identical layer groups, both containing a mask with a larger image.
2. One is a regular layer group, the second one turned into a symbol.
3. I make both layers scrollable within Framer
4. Only the regular layer group is scrollable:

![symbolsmasking](https://cloud.githubusercontent.com/assets/869934/6190400/ecabbb70-b3a1-11e4-9874-4e17131f3421.png)

**Comment by koenbok** [comment]
TLDR; masked images in symbols get cropped to mask size.

**Comment by koenbok** [comment]
Fixed verified.
